Dockerfile

```diff
@@ -6,10 +6,8 @@ RUN make build
 FROM registry.svc.ci.openshift.org/openshift/origin-v4.0:base
 COPY --from=builder /ingress-operator/ingress-operator /usr/bin/
 COPY manifests /manifests
-RUN useradd ingress-operator
-USER ingress-operator
```

Comment:
This new user should be unnecessary, and by using, the installer (as an image consumer) must override the user when doing things like running `render` with a volume mounted output directory (because `ingress-operator` user doesn't have permissions to write to the volume mount point).

Although I can work around it in the installer, if we have no reason to run as a new user, removing the `useradd` makes things more consistent with the other images used for the same purpose.